<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天气预报(测试版)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-11</x>
      <y>-11</y>
      <width>831</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{
	border-image: url()
}

QLabel{
	font: 25 10pt &quot;微软雅黑&quot;;
	border-radius: 4px;
	background-color: rg ba( 60, 60, 60, 100 );
	color: rg b( 255, 255, 255 );
}</string>
    </property>
    <widget class="QWidget" name="widget_1" native="true">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>229</y>
       <width>341</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_8" rowstretch="1,2">
      <item row="0" column="0">
       <widget class="QWidget" name="widget_14" native="true">
        <widget class="QLabel" name="ganmao">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>311</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_15" native="true">
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="1" column="1">
          <widget class="QWidget" name="widget_19" native="true">
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>51</width>
              <height>51</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="Figure.qrc">:/new/prefix1/Figure/aqi.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="TotalQuality">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>0</y>
              <width>91</width>
              <height>51</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	font: 25 9pt &quot;微软雅黑&quot;;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QWidget" name="widget_16" native="true">
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>3</x>
              <y>0</y>
              <width>51</width>
              <height>51</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="Figure.qrc">:/new/prefix1/Figure/wind.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>30</y>
              <width>51</width>
              <height>51</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="Figure.qrc">:/new/prefix1/Figure/wind.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="TotalFxFl">
            <property name="geometry">
             <rect>
              <x>63</x>
              <y>1</y>
              <width>81</width>
              <height>51</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QWidget" name="widget_17" native="true">
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>51</width>
              <height>51</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="Figure.qrc">:/new/prefix1/Figure/pm25.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="PM25">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>0</y>
              <width>91</width>
              <height>51</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QWidget" name="widget_18" native="true">
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>51</width>
              <height>51</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="Figure.qrc">:/new/prefix1/Figure/humidity.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="Totalshidu">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>0</y>
              <width>81</width>
              <height>51</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>349</x>
       <y>59</y>
       <width>461</width>
       <height>401</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="2,2,2,3,3,2">
      <item row="2" column="0">
       <widget class="QWidget" name="widget_7" native="true">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="aqi0">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="aqi4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="aqi1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="aqi2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="aqi3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="aqi5">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QWidget" name="HighCurve" native="true">
        <widget class="QLabel" name="lblHighCurve">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>441</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="fx0">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="fx4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="fx3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="fx1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="fx5">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="fx2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="fl0">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="fl1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="fl2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="fl3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="fl4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="fl5">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QWidget" name="LowCurve" native="true">
        <widget class="QLabel" name="lblLowCurve">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>441</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_9" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QLabel" name="week1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="week2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="date0">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="date2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="week3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="week4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="week0">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="date1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="week5">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="date3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="date4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="date5">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="icon0">
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="type0">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="icon2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="icon4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="icon3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="type1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="icon1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QLabel" name="icon5">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="type2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="type3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="type4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QLabel" name="type5">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>60</y>
       <width>361</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6" columnstretch="2,3">
      <item row="0" column="0">
       <widget class="QWidget" name="widget_10" native="true">
        <widget class="QLabel" name="TotalTypeIcon">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>91</width>
           <height>91</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widget_11" native="true">
        <layout class="QGridLayout" name="gridLayout_7" rowstretch="2,1">
         <item row="0" column="0">
          <widget class="QWidget" name="widget_12" native="true">
           <widget class="QLabel" name="city">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>30</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	font: 25 16pt &quot;微软雅黑&quot;;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="temperature">
            <property name="geometry">
             <rect>
              <x>3</x>
              <y>0</y>
              <width>91</width>
              <height>81</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	font: 25 30pt &quot;微软雅黑&quot;;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
           </widget>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QWidget" name="widget_13" native="true">
           <widget class="QLabel" name="typelowhigh">
            <property name="geometry">
             <rect>
              <x>3</x>
              <y>1</y>
              <width>181</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	font: 25 14pt &quot;微软雅黑&quot;;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>-11</x>
       <y>9</y>
       <width>861</width>
       <height>51</height>
      </rect>
     </property>
     <widget class="QLabel" name="TotalDate">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>0</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
        <weight>3</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 25 16pt &quot;微软雅黑&quot;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchcity">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="search">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>12</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Figure.qrc">
        <normaloff>:/new/prefix1/Figure/search.png</normaloff>:/new/prefix1/Figure/search.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="closeButton">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Figure.qrc">
        <normaloff>:/new/prefix1/Figure/close.png</normaloff>:/new/prefix1/Figure/close.png</iconset>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionWeatherAI"/>
    <addaction name="actionWeatherAI_Python"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionWeatherAI">
   <property name="text">
    <string>WeatherAI(C++)</string>
   </property>
  </action>
  <action name="actionWeatherAI_Python">
   <property name="text">
    <string>WeatherAI(Python)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Figure.qrc"/>
 </resources>
 <connections/>
</ui>
